api/spec/controllers/mno_enterprise/jpi/v1/admin/subscription_events_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+module MnoEnterprise
+  describe Jpi::V1::Admin::SubscriptionEventsController, type: :controller do
+    include MnoEnterprise::TestingSupport::SharedExamples::JpiV1Admin
+
+    render_views
+    routes { MnoEnterprise::Engine.routes }
+    before { request.env["HTTP_ACCEPT"] = 'application/json' }
+
+    before(:each) do
+      Settings.merge!(dashboard: { marketplace: {provisioning: true } })
+      Rails.application.reload_routes!
+    end
+
+    #===============================================
+    # Assignments
+    #===============================================
+    let(:user) { build(:user, :admin) }
+    let!(:current_user_stub) { stub_user(user) }
+
+    let(:organization) { build(:organization) }
+    let(:subscription) { build(:subscription) }
+    let(:subscription_event) { build(:subscription_event, subscription: subscription) }
+
+    #===============================================
+    # Specs
+    #===============================================
+    before { sign_in user }
+
+    describe 'GET #index' do
```

Comment:
Also test the shared behavior with `it_behaves_like 'a jpi v1 admin action'` (same for `#show`)